docs/book/v3/middleware.md

```diff
@@ -69,6 +84,13 @@ another handler_.
 > Stratigility supports only [PSR-15](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-15-request-handlers.md)
 > middleware.
 
+> ### HTTP Handler Runner
+>
+> The above example, and the one on the [usage page](usage.md), makes use of the
+> [laminas/laminas-httphandlerrunner](https://docs.laminas.dev/laminas-httphandlerrunner/)
+> package, which provides a flexible way to run applications based on PSR-15
+> request handlers.
+
```

Comment:
This means the page for installation and the Composer command has to be updated:

https://docs.laminas.dev/laminas-stratigility/v3/install/

Reply:
It doesn't.

You can use Stratigility without laminas-httphandlerrunner. I'm just using it to demonstrate one possible way to run your middleware application.

Reply:
Okay then the example needs an installation otherwise it won't work if a user wants to test it.

See: https://docs.laminas.dev/laminas-i18n/translator/events/#example

Reply:
Done!